lib/client-assets.php

```diff
@@ -73,6 +73,12 @@ function gutenberg_get_script_polyfill( $tests ) {
  * @since 0.1.0
  */
 function gutenberg_register_scripts_and_styles() {
+	// We need to return early when this GET variable is set. This happens when the editor loads the metaboxes and for
+	// this case we don't need to register scripts and styles.
```

Comment:
Just curious how why we do not want to register the scripts and styles in this case? I mean why registering the scripts/styles breaks the desired output. If we register something it's not enqueued directly unless I'm missing something?

Reply:
You are right, my bad. This check should be moved where the scripts are enqueued.